env0/utils.go

```diff
@@ -0,0 +1,141 @@
+package env0
```

Comment:
Everything else in this PR is straightforward. The significant/big change is this file.
In the long run, it will reduce code clutter.